src/globals/mixins.scss

```diff
@@ -7,6 +7,17 @@
 }
 
 // Layout + spacing
+@mixin flex-container-row() {
+  display: -webkit-box;
```

Comment:
I don't think these vendor specific selectors are useful these days? Let's avoid if it's not expanding browser support meaningfully so we don't make the components heavier than they need to be.